src/System.Management.Automation/cimSupport/cmdletization/xml/cmdlets-over-objects.xmlSerializer.autogen.cs

```diff
@@ -35,7 +35,7 @@ public void Write51_ClassMetadata(object o)
             WriteStartDocument();
             if (o == null)
             {
-                WriteNullTagLiteral(@"ClassMetadata", @"");
+                WriteNullTagLiteral(@"ClassMetadata", string.Empty);
```

Comment:
If this is an autogen'ed file should you be changing it or the generator?

Reply:
The files is not regenerated and already was changed manually. Seems we should rename the files and remove "autogen" from file names.

Reply:
And remove the comments saying "this is an autogen'ed  file, don't edit"

Reply:
Tracking issue #6974